example_project/example_project/settings/base.py

```
@@ -227,10 +227,20 @@
+# Django default storage and django-dbbackup
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.StaticFilesStorage",
+    },
+    "dbbackup": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+        "OPTIONS": {
+            "location": env.str("DBBACKUP_STORAGE_LOCATION", default="/tmp/backups-example")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/backups-example"

(S108)